src/features/unicode/contextCheck/variationSequenceCheck.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Unicode Variation Sequence context checkers
+ */
+
+function isVariationSequenceSelector(char) {
+    if (char === null) return false;
+    const charCode = char.codePointAt(0);
+    return (
+        // Mongolian Variation Selectors
+        (charCode >= 0x180B && charCode <= 0x180D) ||
+        // Generic Variation Selectors
+        (charCode >= 0xFE00 && charCode <= 0xFE0F) ||
```

Comment:
Are we accounting for variation selectors on emoji?

Reply:
All combinations in https://www.unicode.org/Public/UCD/latest/ucd/emoji/emoji-variation-sequences.txt use the variation selector FE0E, so they are included here. UVSes only handle combinations of one base character and a variation selector. More complex emoji variations using zero-width non-joiner etc. are something different and not covered by this.

Reply:
from wikipedia:

They are currently used to specify standardized variation sequences for mathematical symbols, [emoji](https://en.wikipedia.org/wiki/Emoji) symbols, ['Phags-pa](https://en.wikipedia.org/wiki/%27Phags-pa_script) letters, and CJK unified ideographs corresponding to [CJK compatibility ideographs](https://en.wikipedia.org/wiki/CJK_compatibility_ideographs). At present only standardized variation sequences with VS1, VS2, VS3, VS15 and VS16 have been defined; VS15 and VS16 are reserved to request that a character should be displayed as text or as an [emoji](https://en.wikipedia.org/wiki/Emoji) respectively.

Reply:
or am i reading something different

Reply:
We'll support all current and future selectors in the three ranges according to
https://www.unicode.org/glossary/#variation_selector and https://freetype.org/freetype2/docs/reference/ft2-glyph_variants.html  But maybe I just don't understand what you're getting at? 😄

Reply:
OK, nevermind was just making sure.